new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDedicatedMode.java

```diff
@@ -0,0 +1,391 @@
+   *                   JUnit engine parameter resolution mechanism.
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    logger = getLogger();
+    // get a unique operator namespace
+    logger.info("Getting a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    // in the dedicated mode, the operator only manages domains in the operator's own namespace
+    domain1Namespace = opNamespace;
+
+    // get a new unique domainNamespace
+    logger.info("Assigning a unique namespace for Domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domain2Namespace = namespaces.get(1);
+
+    // Variables for Operator
+    opServiceAccount = opNamespace + "-sa";
+    opHelmParams =
+        new HelmParams().releaseName(OPERATOR_RELEASE_NAME)
+            .namespace(opNamespace)
+            .chartDir(OPERATOR_CHART_DIR);
+  }
+
+  @AfterAll
+  public void tearDownAll() {
+  }
+
+  /**
+   * Set "dedicated" to true and set the domainNamespaces to something that does not contain the operator's namespace,
+   * make sure that the domains in the operator's target namespaces do not come up
+   * because when dedicated is set to true, the operator's domainNamespaces value is ignored.
+   * 1) Install an Operator with namespace=op-ns, domainNamespaces=wls-ns
+   *    and domainNamespaceSelectionStrategy=Dedicated.
+   * 2) Verify the Operator is up and running.
+   * 3) Create WebLogic Domain its namespace=wls-ns.
+   * 4) Verify that domain whose namespace = wls-ns does not come up.
+   */
+  @Test
+  @Order(1)
+  @DisplayName("Set dedicated to true and verify that a domain not deployed in operator's namespace doesn't come up")
+  public void testDedicatedModeDiffNamespace() {
+    // install and verify operator
+    logger.info("Installing and verifying operator");
+    installAndVerifyOperator(opNamespace, opNamespace + "-sa",
+        true, 0, opHelmParams, domainNamespaceSelectionStrategy,
+        false, domain2Namespace);
+
+    // create and verify the domain
+    logger.info("Creating and verifying model in image domain");
+    createDomain(domain2Namespace);
+
+    verifyDomainNotRunning(domain2Namespace);
+  }
+
+  /**
+   * Set "dedicated" to true, make sure that the domains deployed in the operator's namespace come up.
+   * 1) Using the Operator's namespace, op-wls-ns, create a WebLogic Domain
+   * 2) Verify that the WebLogic domain whose namespace = op-wls-ns comes up.
+   */
+  @Test
+  @Order(2)
+  @DisplayName("Set dedicated to true and verify that the domain deployed in the operator's namespace comes up")
+  public void testDedicatedModeSameNamespace() {
+    // create and verify the domain
+    logger.info("Creating and verifying model in image domain");
+    createDomain(domain1Namespace);
+    verifyDomainRunning(domain1Namespace);
+  }
+
+  /**
```

Comment:
Need more description of the usecase. It is too generic. If we generate javadoc, it MUST provide the exact detail of the usecase.

Reply:
That's exactly use case. I don't know what else to give :(

Reply:
Then you can merge the test method to another test method to provide a complete e2e scenario including scaling.